client/api/volume/v1alpha1/api.proto

```diff
@@ -0,0 +1,69 @@
+syntax = "proto3";
+
+package v1alpha1;
+
+service Volume {
+    // ListVolumesOnDisk returns the volume IDs (in \\.\Volume{GUID} format) for
+    // all volumes on a Disk device
+    rpc ListVolumesOnDisk(ListVolumesOnDiskRequest) returns (ListVolumesOnDiskResponse) {}
+    // MountVolume mounts the volume at the requested global staging path
+    rpc MountVolume(MountVolumeRequest) returns (MountVolumeResponse) {}
+    // DismountVolume gracefully dismounts a volume
+    rpc DismountVolume(DismountVolumeRequest) returns (DismountVolumeResponse) {}
+    // IsVolumeFormatted checks if a volume is formatted with NTFS
+    rpc IsVolumeFormatted(IsVolumeFormattedRequest) returns (IsVolumeFormattedResponse) {}
+    // FormatVolume formats a volume with the provided file system
+    rpc FormatVolume(FormatVolumeRequest) returns (FormatVolumeResponse) {}
+    // ResizeVolume performs resizing of the partition and file system for a block based volume
+    rpc ResizeVolume(ResizeVolumeRequest) returns (ResizeVolumeResponse) {}
+}
+message ListVolumesOnDiskRequest {
+    // Disk device ID of the disk to query for volumes
+    string disk_id = 1;
+}
+message ListVolumesOnDiskResponse {
+    // Volume device IDs of volumes on the specified disk
```

Comment:
Can you clarify the `\\.\Volume{GUID} format` ID format in the comments wherever volume_id[s] are involved?

Reply:
Sure, will do.